lib/screens/settings/about.dart

```diff
@@ -0,0 +1,126 @@
+        ),
+      ),
+    );
+  }
+}
+
+_launchURL() async {
+  const url = 'https://github.com/anitab-org/mentorship-flutter';
+  if (await canLaunch(url)) {
+    await launch(url);
+  } else {
+    print("Could not launch");
+  }
+}
+
+@override
+Widget _buildMainCard(BuildContext context) {
+  return Card(
+    semanticContainer: true,
+    clipBehavior: Clip.antiAlias,
+    child: Column(
+      children: <Widget>[
+        Image.asset("assets/images/mentorship_system_logo.png"),
+        Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: ExpandablePanel(
+            header: Text(
+              "Who are we?",
+              style: TextStyle(
+                fontWeight: FontWeight.bold,
+                fontSize: 22,
+              ),
+            ),
+            collapsed: Text("AnitaB.org is a global nonprofit organization"),
+            expanded: Text(
+                "AnitaB.org is a global nonprofit organization based in Belmont, California. Founded by computer scientists Anita Borg and Telle Whitney, the institute's primary aim is to recruit, retain, and advance women in technology."),
+          ),
+        ),
+      ],
+    ),
+  );
+}
+
+@override
+Widget _buildContributingCard(BuildContext context) {
+  return Card(
+    child: Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: ExpandablePanel(
+        header: Text(
+          "Contributing to Mentorship",
+          style: TextStyle(
+            fontWeight: FontWeight.bold,
+            fontSize: 22,
+          ),
+        ),
+        collapsed: Text("Head on over to our Github to contribute"),
+        expanded: Container(
+          height: 40,
+          width: 100,
+          child: RaisedButton(
+            color: Color.fromRGBO(36, 41, 46, 1),
+            child: Row(
+              children: <Widget>[
+                Image.asset(
+                  "assets/images/Github.png",
+                  width: 20,
+                  height: 25,
+                ),
+                SizedBox(
+                  width: 5,
+                ),
+                Text(
+                  "Github",
+                  style: TextStyle(color: Colors.white),
+                ),
+              ],
+            ),
+            onPressed: _launchURL,
+          ),
+        ),
+      ),
+    ),
+  );
+}
+
+@override
+Widget _buildGeneralCard(BuildContext context) {
+  return Card(
+    child: Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: ExpandablePanel(
+        header: Text(
+          "What is the Mentorship System",
+          style: TextStyle(
+            fontWeight: FontWeight.bold,
+            fontSize: 22,
+          ),
+        ),
+        collapsed: Text("Mentorship System is a cross platform application"),
+        expanded: Text(
+          "Mentorship System is a cross platform application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period of time.",
```

Comment:
Is there any option for your UI to show like "..." to let people know they can expand that? I don't know if it's possible here

Reply:
we can add ... in the short description?

Reply:
cool that sounds good!

Reply:
also the short description says very little of the application mission :P can you add a short description to be the description in the repository? @Techno-Disaster

Reply:
@isabelcosta Ok sure, but as i said in the comments above and on zulip, this data is just place holder and we will put the actual data during release. I'm not sure what "application mission" I have to put there.

is this ok
> Mentorship system allows women in tech ...

and in expanded description

> Mentorship System is an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period of time.

Reply:
Yes that is fine. Also we can leave this for content team in the future. I am fine with this PR for now :)